packages/flutter/test/widgets/autocomplete_test.dart

```diff
@@ -35,7 +36,7 @@ void main() {
     'koala',
     'lemur',
     'mouse',
-    'northern white rhinocerous',
+    'northern white rhinoceros',
```

Comment:
That's attention to detail :-)

Reply:
That's an IDE that points out typos and an OCD engineer... 😄